.gitignore

```diff
@@ -13,3 +13,4 @@
 .vscode/
 .DS_Store
 **/.ipynb_checkpoints/*
+**/data/*
```

Comment:
Where do "data" folders get generated or why is this being added to the .gitignore?

Reply:
This is for silencing a folder that I created for IRA analysis. Maybe it's worth discussing how to handle branches that would not get merged back into resstock.